holidays/be.yaml

```diff
@@ -12,3 +11,33 @@ PH:
   - {'name': 'Allerheiligen - Toussaint', 'fixed_date': [11, 1]}
   - {'name': 'Wapenstilstand - Armistice', 'fixed_date': [11, 11]}
   - {'name': 'Kerstmis - Noël', 'fixed_date': [12, 25]}
+
+# Source: https://nl.wikipedia.org/wiki/Schoolvakantie
```

Comment:
The `_nominatim_url` is needed. You already had it, but with German coordinates as it seems. See https://github.com/opening-hours/opening_hours.js/tree/master/holidays for details.

Reply:
I've added in the style of the ch.yaml which just requests 'Belgium'